actors/miner/src/types.rs

```diff
@@ -127,6 +127,39 @@ pub struct ProveCommitSectorParams {
     pub proof: RawBytes,
 }
 
+// Note no UnsealedCID because it must be "zero" data.
+#[derive(Clone, Debug, Eq, PartialEq, Serialize_tuple, Deserialize_tuple)]
+pub struct SectorNIActivationInfo {
+    pub sealing_number: SectorNumber,
+    pub sealer_id: ActorID,
+    pub sealed_cid: Cid, // CommR
+    pub sector_number: SectorNumber,
```

Comment:
```suggestion
    pub sealing_number: SectorNumber, // Sector number used to generate replica id
    pub sealer_id: ActorID,           // Must be set to ID of receiving actor for now
    pub sealed_cid: Cid,              // CommR
    pub sector_number: SectorNumber,  // Unique id of sector in actor state
```

just for consistency with the fip, for now, we can improve later but best have some rather than none